### ISSUE TITLE
Freecam Not Rendering Correctly 1.21 Lunar Client

### ISSUE THREAD

**Issue by polarbearClof** [MinecraftFreecam/Freecam#246]
### Description of the issue

If you try to use freecam with lunar client the terrain wont render correctly, sometimes it renders if youre close open spaces sometimes it doesnt.

https://github.com/user-attachments/assets/449c4153-d06d-4ca7-a27a-81c112bf67b6

https://github.com/user-attachments/assets/fa4a26a1-6a7a-4e0a-bba2-ab6e2c5c6ac2

### Expected behavior

I expected Freecam to show all terrain when activated

### Steps to reproduce

1. Open Lunar Client using Fabric module 1.21
2. Go on a world / server
3. Activate Freecam (NOT in spectator mode)

### Log output

```shell
[renderer.log](https://github.com/user-attachments/files/16788642/renderer.log)
[renderer.old.log](https://github.com/user-attachments/files/16788643/renderer.old.log)
```

### How prevalent is the issue?

Every time i try to use activate Freecam

### Freecam version

i tried both 1.3.0 and 1.3.0-beta1

### Are you using "Modrinth Edition"?

No

### Minecraft version

1.21

### Modding platform

- [X] Fabric
- [ ] Neoforge
- [ ] Forge

**Comment by polarbearClof** [comment]
Correct logs:
[renderer.old.log](https://github.com/user-attachments/files/16788658/renderer.old.log)
[renderer.log](https://github.com/user-attachments/files/16788659/renderer.log)

**Comment by Devsider** [comment]
I'm having similar issues using Fabric.

**Comment by Devsider** [comment]
Did some Testing, only happens when also using the Iris Mod at the same Time. No Issues without it.

**Comment by MattSturgeon** [comment]
@Devsider sounds like you're running into #205. There's a fix available upstream in Iris (not sure if its made it into a release yet though). See https://github.com/MinecraftFreecam/Freecam/issues/205#issuecomment-2408710856

**Comment by Devsider** [comment]
> @Devsider sounds like you're running into #205. There's a fix available upstream in Iris (not sure if its made it into a release yet though). See [#205 (comment)](https://github.com/MinecraftFreecam/Freecam/issues/205#issuecomment-2408710856)

Yep! The Problem was solved with the solution from the comment! Thanks a lot.
